docs/src/jekyll/index.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: home
+technologies:
+ - scala: ["Scala", "Fetch is written in Scala and supports both Scala (JVM) and Scala.js (JavaScript environments)"]
+ - cats: ["Cats", "Fetch uses cats' Free Monad implementation as well as some of its data types."]
+ - fp: ["Functional Programming", "Fetch is implemented using the Free Monad and Interpreter pattern."]
+---
+
+## Installation
+
+Add the following dependency to your project's build file.
+
+```scala
+"org.fortysevendeg" %%% "fetch" %% "0.1.0"
```

Comment:
This should be "com.fortysevendeg". also I recall there was an internal proposal to change our internal packages to something shorter.